server/hedley/modules/custom/hedley_person/hedley_person.module

```diff
@@ -1026,6 +1043,7 @@ function hedley_person_create_basic_node($type, $author_id = FALSE) {
   // Mark node as imported, to be able to identify it at hook_insert().
   $node->imported = 1;
   $node = node_submit($node);
+  node_save($node);
```

Comment:
@AronNovak 
Saving node is not a job of this function.
Throughout the code, whenever node is created, wrapper is used to populate node data.
Node is saved using the wrapper. 
This functionality prvents problems with `node_insert / node_update` hooks that expect to see data on node.

Please revert this change (and try-catch clauses in PR), or, lets discuss...

Reply:
I took a closer look, as I didn't remember the root cause, and that's why:
![image](https://user-images.githubusercontent.com/114076/144201941-ac9c90e5-9496-4ed5-9b63-9ebf507290aa.png)
It was just a result of bad move(s) in Git, I revert everything what's not related to reporting.

Reply:
@AronNovak 

I see. In this case, please make sure that this PR contains only intended changes, then MWG.